src/__tests__/slug.spec.js

```diff
@@ -107,6 +107,14 @@ describe('slugizoid', () => {
   describe('creation', () => {
     const expectedSlug = 'pull_request';
 
+    it('should remove non-alphanumeric characters', () => {
+      // Arrange
+      const slug = slugizoid('pull)*&^%$#@!(=+}{][]}/\'"`;:.,request');
```

Comment:
You should add the following characters for coverage completeness:
- whitespace (\s)
- dash
- underscore